src/pages/home/home.ts

```diff
@@ -26,12 +26,16 @@ export class HomePage {
           break;
         }
         case 'Edit': {
-          console.log('Edit');
+          this.navCtrl.push(ItemPage, {
```

Comment:
I would feel more comfortable with this if it used an [enum](https://www.typescriptlang.org/docs/handbook/enums.html) instead of passing strings that act as labels for the different actions.